Processes/Docs-Contribution-Process.md

```diff
@@ -0,0 +1,25 @@
+
+
+# Docs Contribution Process
+
+## Overview
+This document provides instructions for content contributors to the [EOS Documentation Portal](https://docs.eosnetwork.com).
+
+Create a pull request (PR) to submit content to the Docs Portal. The DevRel team will check the accuracy of the content.
+
+
+
+
+### Stages of Submission Process
+1. Create branch
```

Comment:
These steps are too detailed. You should introduce/refer to the other documents in this `Processes` folder. Remember that this file also acts as an index to the `Processes` folder. Something like:

1\. First, learn about our \[doc templates\]\(link to doc\).
2\. Then, follow the guidelines to \[write your content\]\(link to doc\).
3\. etc...